SMLHelper/Assets/PdaItem.cs

```diff
@@ -58,7 +58,7 @@ internal PdaItem(string classId, string friendlyName, string description)
         /// <summary>
         /// This provides the <see cref="TechData"/> instance used to designate how this item is crafted or constructed.
         /// </summary>
-        protected abstract TechData GetBlueprintRecipe();
+        protected abstract RecipeData GetBlueprintRecipe();
```

Comment:
**TODO** Update with conditional compilation. 
Update XML documentation to match code.
```CS
#if SUBNAUTICA 
/// <summary>
/// This provides the <see cref="TechData"/> instance used to designate how this item is crafted or constructed.
/// </summary>
protected abstract TechData GetBlueprintRecipe();
#elif BELOWZERO
/// <summary>
/// This provides the <see cref="RecipeData"/> instance used to designate how this item is crafted or constructed.
/// </summary>
protected abstract RecipeData GetBlueprintRecipe();
#endif
```